src/dynarec/rv64/rv64_emitter.h

```diff
@@ -967,9 +967,14 @@ f28–31  ft8–11  FP temporaries                  Caller
 
 // Single-bit Extract (Register), s0 can be the same as rs2
```

Comment:
I think the `s0 can be the same as rs2` statement can be a bit confusing, maybe remove it.

Reply:
Done